source/appModules/eclipse.py

```diff
@@ -15,6 +15,8 @@
 import keyboardHandler
 from scriptHandler import script
 
+category_title = "Eclipse"

```

Comment:
The other categories names in NVDA are defined as constants, i.e. UPPERCASE_NAME, and begin with "SCRCAT_", e.g. SCRCAT_PYTHON_CONSOLE or SCRCAT_SPEECH. They are also translatable. Even if these are application names, I think that they need to be translatable anyway.

```suggestion
# Translators: The name of a category of NVDA commands.
categorySCRCAT_ECLIPSE = _("Eclipse")
```